modules/users/client/components/AboutMe.component.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { NamespacesConsumer } from 'react-i18next';
+import '@/config/lib/i18n';
+import PropTypes from 'prop-types';
+import { limitTo, plainTextLength } from './utils/filters';
+
+export default function AboutMe({ profile, profileDescriptionToggle, user, appSettings }) {
+  return (<NamespacesConsumer ns={['user-profile']}>{ (t) => (<>
+    <section className="panel panel-default">
+      <header className="panel-heading">
+        {t('About me')}
+      </header>
+      <div className="panel-body">
+        {/* Short descriptions */}
+        {(profile.description.length < 2400 || profileDescriptionToggle) &&
+        <div dangerouslySetInnerHTML={{ __html: profile.description }} ></div>}{/* TODO Can we trust this HTML?*/}
+
+        {/* Long descriptions */}
+        {profile.description.length >= 2400 && !profileDescriptionToggle &&
+        <div
+          className="panel-more-wrap" ng-click="profileCtrl.profileDescriptionToggle=true">
+          <div className="panel-more-excerpt" setInnerHTML={limitTo(profile.description, 2400)}> </div>
+          <div className="panel-more-fade">{t('Show more...')}</div>
```

Comment:
Clicking `Show more...` doesn't work for me. It might be related to the `setInnerHTML` issue above; or to failing logic related to `profileDescriptionToggle`.